ember-flight-icons/tests/dummy/app/templates/index.hbs

```diff
@@ -27,7 +27,7 @@
   </div>
 
   <div class="relative group bg-white p-6 focus-within:ring-2 focus-within:ring-inset focus-within:ring-brand">
-    <a href="" class="ds-a ds-h3 focus:outline-none hover:underline">
+    <a href="design/#h2-contributing" class="ds-a ds-h3 focus:outline-none hover:underline" rel="external">
```

Comment:
This will need to be updated when #241 gets merged. Happy to adjust in that branch though as I'm sure this will get merged before that one.